be/src/storage/inverted/clucene/clucene_inverted_writer.cpp

```diff
@@ -121,8 +122,25 @@ class CLuceneInvertedWriterImpl : public CLuceneInvertedWriter {
         if constexpr (is_string_type(field_type)) {
             auto* _val = (Slice*)values;
             for (int i = 0; i < count; ++i) {
-                new_fulltext_field(_val->data, _val->size);
-                _index_writer->addDocument(_doc.get());
+                const char* s = _val->data;
+                size_t size = _val->size;
+                if (size > _wstring_buffer.size()) {
+                    _wstring_buffer.resize(size);
```

Comment:
`reserve`?

Reply:
`reserve `may make the size() unchanged. We can not get the current _wstring_buffer size in this case. But we need to change it in every loop.

Reply:
use reserve instead